app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RecipesController < ApplicationController
+
+  def show 
+    @recipe = Recipe.find(params[:id])
+    # @ingredient = Ingredient.find(params{:id})
```

Comment:
Keep in mind that you have access to a recipe's ingredients as an attribute (through the associations with the many-to-many set up). So, in the view, you could call `recipe.ingredients` and iterate over those ingredients to display the list